nexus-repository-p2/src/test/java/org/sonatype/nexus/repository/p2/internal/P2BrowseNodeGeneratorTest.java

```diff
@@ -15,13 +15,15 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.stream.Collectors;
 
 import org.sonatype.nexus.repository.browse.BrowsePaths;
 import org.sonatype.nexus.repository.browse.BrowseTestSupport;
 import org.sonatype.nexus.repository.p2.internal.util.P2PathUtils;
 import org.sonatype.nexus.repository.storage.Asset;
 import org.sonatype.nexus.repository.storage.Component;
 
+import org.junit.Assert;
 import org.junit.Test;
 
 public class P2BrowseNodeGeneratorTest
```

Comment:
add a test for a repo without remote, for metadata...

Reply:
Done